packages/pwa-kit-react-sdk/package.json

```diff
@@ -1,79 +1,79 @@
-    "@loadable/component": "^5.15.0",
-    "react": ">=16.14 || <18",
-    "react-dom": ">=16.14 || <18",
-    "react-helmet": "6",
-    "react-router-dom": "^5.1.2"
-  }
+    "name": "pwa-kit-react-sdk",
+    "version": "2.7.2",
+    "description": "A library that supports the isomorphic React rendering pipeline for Commerce Cloud Managed Runtime apps",
+    "engines": {
+        "node": "^14.0.0 || ^16.0.0 || ^18.0.0",
+        "npm": "^6.14.4 || ^7.0.0 || ^8.0.0 || ^9.0.0"
+    },
+    "files": [
+        "CHANGELOG.md",
+        "LICENSE",
+        "scripts",
+        "ssr",
+        "utils"
+    ],
+    "publishConfig": {
+        "directory": "dist"
+    },
+    "scripts": {
+        "prepare": "npm run build",
+        "build": "cross-env NODE_ENV=production internal-lib-build build",
+        "build:watch": "watch 'npm run build' src/",
+        "lint": "npm run lint:js",
+        "lint:fix": "npm run lint:js -- --fix",
+        "lint:js": "internal-lib-build lint \"**/*.{js,jsx}\"",
+        "test": "internal-lib-build test",
+        "test:watch": "npm test -- --watch",
+        "test:inspect": "node --inspect-brk jest --runInBand",
+        "version": "node ./scripts/version.js",
+        "format": "internal-lib-build format \"**/*.{js,jsx}\""
+    },
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/SalesforceCommerceCloud/pwa-kit.git"
+    },
+    "homepage": "https://github.com/SalesforceCommerceCloud/pwa-kit/tree/develop/packages/pwa-kit-react-sdk#readme",
+    "bugs": {
+        "url": "https://github.com/SalesforceCommerceCloud/pwa-kit/issues"
+    },
+    "author": "cc-pwa-kit@salesforce.com",
+    "license": "SEE LICENSE IN LICENSE",
+    "dependencies": {
+        "@loadable/babel-plugin": "^5.15.3",
+        "@loadable/server": "^5.15.3",
+        "@loadable/webpack-plugin": "^5.15.2",
+        "@tanstack/react-query": "^4.28.0",
+        "cross-env": "^5.2.1",
+        "event-emitter": "^0.3.5",
+        "hoist-non-react-statics": "^3.3.2",
+        "prop-types": "^15.8.1",
+        "pwa-kit-runtime": "^2.7.2",
+        "react-ssr-prepass": "^1.5.0",
+        "react-uid": "^2.3.2",
+        "serialize-javascript": "^6.0.1",
+        "svg-sprite-loader": "^6.0.11",
+        "watch": "1.0.2"
+    },
+    "devDependencies": {
+        "@loadable/component": "^5.15.3",
+        "@wojtekmaj/enzyme-adapter-react-17": "^0.8.0",
+        "enzyme": "^3.11.0",
+        "internal-lib-build": "^2.7.2",
+        "node-html-parser": "^3.3.6",
+        "react": "^17.0.2",
+        "react-dom": "^17.0.2",
+        "react-helmet": "5.2.1",
+        "react-router-dom": "^5.3.4",
+        "regenerator-runtime": "^0.13.11",
+        "sinon": "^13.0.2",
+        "supertest": "^4.0.2"
+    },
+    "peerDependencies": {
+        "@loadable/component": "^5.15.0",
+        "react": ">=16.14 || <18",
+        "react-dom": ">=16.14 || <18",
+        "react-helmet": "6",
+        "react-router-dom": "^5.1.2"
+    }
```

Comment:
@shethj all of the `package.json` files should have `tabWidth: 2` but it looks like they've been saved with `tabWidth: 4` which means that when npm regenerates them later (with `tabWidth: 2`) it creates extra headaches

Reply:
Added an override in all `prettierrc.yaml` files to set `tabWidth: 2` for `package.json` files